src/node/NodeLoaderCompiler.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {LoaderCompiler} from '../runtime/LoaderCompiler.js';
```

Comment:
Yes, that should work. The other distinction of `src/node/` is that it is shipped in the npm package. But this file will be compiled in to `bin/traceur.js`.

Done in next commit